sdk/ml/azure-ai-ml/azure/ai/ml/entities/_component/component.py

```diff
@@ -530,6 +533,9 @@ def _resolve_local_code(self):
                     # that might result in unexpected issues and also break related tests.
                     # therefore we copy deeper to avoid this break.
                     src_path = Path(self._base_path) / code
-                    dst_path = Path(tmp_dir) / src_path.name
+                    # .name will return empty string for UNC drive names
+                    # so we need src_path.resolve() here to avoid empty string
+                    # that leads to FileExistsError during shutil.copytree
+                    dst_path = Path(tmp_dir) / src_path.resolve().name
```

Comment:
Add a test for this? e.g. when both `Path(self._base_path)` and `code` are `./`

Reply:
added test `test_command_component_code_with_current_folder` on this